client/me/purchases/manage-purchase/index.jsx

```diff
@@ -142,13 +153,17 @@ const ManagePurchase = React.createClass( {
 		upgradesActions.addItems( renewItems );
 
 		page( '/checkout/' + this.props.selectedSite.slug );
-	},
+	}
 
 	renderRenewButton() {
 		const purchase = getPurchase( this.props );
 		const { translate } = this.props;
 
-		if ( ! config.isEnabled( 'upgrades/checkout' ) || ! isRenewable( purchase ) || isExpired( purchase ) || isExpiring( purchase ) || ! getSelectedSite( this.props ) ) {
+		if ( ! config.isEnabled( 'upgrades/checkout' ) ) {
```

Comment:
I wouldn't do it ... at least in this PR :)

I mean, we probably should do it, but we shouldn't remove a single case in a separate PR, but doing a different branch that removes all the use cases at once